### PR TITLE
Add args

### PR DESCRIPTION
**Pull request by TCChenlong**
add some args docs

### CODE REVIEW

File: doc/templates/common_docs.py [hunk 1]
```diff
@@ -4,15 +4,59 @@
...
 common_args_en = """
-    x (Tensor): : the input tensor, it's data type should be float32, float64, int32, int64.
+    x (Tensor): the input tensor, it's data type should be float32, float64, int32, int64.
     name(str, optional): name for the operation (optional, default is None). For more information, please refer to :ref:`api_guide_Name`.
     dtype(str, optional): the data type of the output tensor, can be float32, float64, int32, int64.
-    param_attr(ParamAttr, optional): the parameter attribute for learnable weights(Parameter) of this layer.
+    param_attr(ParamAttr, optional): the parameter attribute for learnable weights(Parameter) of this layer. For more information, please refer to :ref:`api_fluid_ParamAttr`.
+    bias_attr(ParamAttr, optional): the parameter attribute for learnable bias(Bias) of this layer. For more information, please refer to :ref:`api_fluid_ParamAttr`.
+    act (str, optional): Activation to be applied to the output(optional, default is None), such as sigmoid、tanh、relu; For more information, please refer to :ref:`api_guide_activations`.
+    y (Tensor): the input tensor, it's data type should be float32, float64, int32, int64.
```
Comment (jzhang533) [inline doc/templates/common_docs.py]: 这一行挪到x (Tensor)后面吧。

File: doc/templates/common_docs.py [hunk 2]
```diff
@@ -4,15 +4,59 @@
 #
 
 common_args_en = """
-    x (Tensor): : the input tensor, it's data type should be float32, float64, int32, int64.
+    x (Tensor): the input tensor, it's data type should be float32, float64, int32, int64.
     name(str, optional): name for the operation (optional, default is None). For more information, please refer to :ref:`api_guide_Name`.
     dtype(str, optional): the data type of the output tensor, can be float32, float64, int32, int64.
-    param_attr(ParamAttr, optional): the parameter attribute for learnable weights(Parameter) of this layer.
+    param_attr(ParamAttr, optional): the parameter attribute for learnable weights(Parameter) of this layer. For more information, please refer to :ref:`api_fluid_ParamAttr`.
```
Comment (jzhang533) [inline doc/templates/common_docs.py]: 统一一下，参数后面都接个空格吧。

File: doc/templates/common_docs.py [hunk 3]
```diff
@@ -4,15 +4,59 @@
...
+    bias_attr(ParamAttr, optional): the parameter attribute for learnable bias(Bias) of this layer. For more information, please refer to :ref:`api_fluid_ParamAttr`.
+    act (str, optional): Activation to be applied to the output(optional, default is None), such as sigmoid、tanh、relu; For more information, please refer to :ref:`api_guide_activations`.
```
Comment (jzhang533) [inline doc/templates/common_docs.py]: 这行去掉吧，2.0里准备把`act`参数去掉。

File: doc/templates/common_docs.py [hunk 4]
```diff
@@ -4,15 +4,59 @@
...
+    y (Tensor): the input tensor, it's data type should be float32, float64, int32, int64.
+    use_cudnn (bool, optional): Use cudnn kernel or not(optional, default is False), it is valid only when the cudnn library is installed.
```
Comment (jzhang533) [inline doc/templates/common_docs.py]: 这行也去掉吧，2.0里不暴露use_cudnn参数

File: doc/templates/common_docs.py [hunk 5]
```diff
@@ -4,15 +4,59 @@
...
+    label (Tensor): The label of training data, it's data type should be int32, int64.
```
Comment (jzhang533) [inline doc/templates/common_docs.py]: label value corresponding to input.

File: doc/templates/common_docs.py [hunk 6]
```diff
@@ -4,15 +4,59 @@
...
+    learning_rate (Tensor|float): The initial learning rate. If the type is Tensor, it’s a Tensor with shape [1], the data type can be float32 or float64. It also can be set to python float number.
+    axis (int, optional): Axis to compute the input tensor.Default is 0.
```
Comment (jzhang533) [inline doc/templates/common_docs.py]: 这句话有问题吧？

File: doc/templates/common_docs.py [hunk 7]
```diff
@@ -4,15 +4,59 @@
...
+    bias_attr(ParamAttr, optional): the parameter attribute for learnable bias(Bias) of this layer. For more information, please refer to :ref:`api_fluid_ParamAttr`.
+    act (str, optional): Activation to be applied to the output(optional, default is None), such as sigmoid、tanh、relu; For more information, please refer to :ref:`api_guide_activations`.
+    y (Tensor): the input tensor, it's data type should be float32, float64, int32, int64.
+    use_cudnn (bool, optional): Use cudnn kernel or not(optional, default is False), it is valid only when the cudnn library is installed.
+    label (Tensor): The label of training data, it's data type should be int32, int64.
+    learning_rate (Tensor|float): The initial learning rate. If the type is Tensor, it’s a Tensor with shape [1], the data type can be float32 or float64. It also can be set to python float number.
+    axis (int, optional): Axis to compute the input tensor.Default is 0. 
+    out (Tensor, optional): Optional output which can be any created Tensor to store the result of operation. If out is None, a new Tensor will be create to store the result.Default is None.
```
Comment (jzhang533) [inline doc/templates/common_docs.py]: 这个也可以去掉，2.0里去掉了`out`参数。

File: doc/templates/common_docs.py [hunk 8]
```diff
@@ -4,15 +4,59 @@
...
-    param_attr(ParamAttr, optional): the parameter attribute for learnable weights(Parameter) of this layer.
+    param_attr(ParamAttr, optional): the parameter attribute for learnable weights(Parameter) of this layer. For more information, please refer to :ref:`api_fluid_ParamAttr`.
+    bias_attr(ParamAttr, optional): the parameter attribute for learnable bias(Bias) of this layer. For more information, please refer to :ref:`api_fluid_ParamAttr`.
+    act (str, optional): Activation to be applied to the output(optional, default is None), such as sigmoid、tanh、relu; For more information, please refer to :ref:`api_guide_activations`.
+    y (Tensor): the input tensor, it's data type should be float32, float64, int32, int64.
+    use_cudnn (bool, optional): Use cudnn kernel or not(optional, default is False), it is valid only when the cudnn library is installed.
+    label (Tensor): The label of training data, it's data type should be int32, int64.
+    learning_rate (Tensor|float): The initial learning rate. If the type is Tensor, it’s a Tensor with shape [1], the data type can be float32 or float64. It also can be set to python float number.
```
Comment (jzhang533) [inline doc/templates/common_docs.py]: 这句，If后面太啰嗦了。看看怎么精简一下。

File: doc/templates/common_docs.py [hunk 9]
```diff
@@ -4,15 +4,59 @@
...
+    axis (int, optional): Axis to compute the input tensor.Default is 0. 
+    out (Tensor, optional): Optional output which can be any created Tensor to store the result of operation. If out is None, a new Tensor will be create to store the result.Default is None.
+    epsilon (float, optional): Small float added to denominator to avoid dividing by zero. Default is 1e-05.
+    is_test (bool, optional): A flag indicating whether execution is in test phase. Default is False, means not in test phase.
+    shape (Tensor|tuple|list): Shape of the Tensor. If shape is a list or tuple, the elements of it should be integers or Tensors with shape [1]. If shape is Tensor, it should be an 1-D Tensor .
+    keep_dim (bool): Whether to reserve the reduced dimension in the output Tensor. The result tensor will have one fewer dimension than the input unless keep_dim is true. Default is False.
+    seed (int, optional): Random seed. 0 means use a seed generated by the system. Note that if seed is not 0, this operator will always generate the same random numbers every time. Default is 0.
```
Comment (jzhang533) [inline doc/templates/common_docs.py]: seed不常用，先去掉吧。 
paddle没有自己的Random generator。

File: doc/templates/common_docs.py [hunk 10]
```diff
@@ -4,15 +4,59 @@
...
+    filter_size(tuple|list|int): The filter size. If filter_size is a tuple, it must contain two integers, (filter_size_height, filter_size_width). Otherwise, filter_size_height = filter_size_width = filter_size.
```
Comment (jzhang533) [inline doc/templates/common_docs.py]: tf和torch里这个参数都叫`kernel_size`，参考一下他们的写法。

If后面略啰嗦。

File: doc/templates/common_docs.py [hunk 11]
```diff
@@ -4,15 +4,59 @@
...
+    padding (tuple|int): The padding size. If padding is a tuple, it must contain two integers, (padding_height, padding_width). Otherwise, the padding_weight = padding_width = padding. Default is 0.
+    nclude_sublayers (bool, optional): Whether include the sublayers. If True, return list includes the sublayers weights. Default is True.
```
Comment (jzhang533) [inline doc/templates/common_docs.py]: include_sublayers

File: doc/templates/common_docs.py (hunk 1 above)
Comment (TCChenlong) [reply doc/templates/common_docs.py]: 已修改

File: doc/templates/common_docs.py (hunk 2 above)
Comment (TCChenlong) [reply doc/templates/common_docs.py]: 已统一修改

File: doc/templates/common_docs.py (hunk 3 above)
Comment (TCChenlong) [reply doc/templates/common_docs.py]: 已删除

File: doc/templates/common_docs.py (hunk 4 above)
Comment (TCChenlong) [reply doc/templates/common_docs.py]: 已删除

File: doc/templates/common_docs.py (hunk 5 above)
Comment (TCChenlong) [reply doc/templates/common_docs.py]: 已修改

File: doc/templates/common_docs.py (hunk 6 above)
Comment (TCChenlong) [reply doc/templates/common_docs.py]: 已修改

File: doc/templates/common_docs.py (hunk 7 above)
Comment (TCChenlong) [reply doc/templates/common_docs.py]: 已删除

File: doc/templates/common_docs.py (hunk 9 above)
Comment (TCChenlong) [reply doc/templates/common_docs.py]: 好的 已删除

File: doc/templates/common_docs.py (hunk 10 above)
Comment (TCChenlong) [reply doc/templates/common_docs.py]: 好的  已修改

File: doc/templates/common_docs.py (hunk 11 above)
Comment (TCChenlong) [reply doc/templates/common_docs.py]: 已修复

**Review by jzhang533 — APPROVED** [review]
lgtm
